twemproxy_exporter.gemspec

```diff
@@ -1,7 +1,5 @@
 # coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'twemproxy_exporter/version'
+require_relative 'lib/twemproxy_exporter/version'
```

Comment:
Fixed.  I didn't realize that rubygems itself was messing with the path.  Should I revert the executable as well?